etcdctl/ctlv3/ctl.go

```diff
@@ -44,10 +47,14 @@ var (
 		Use:        cliName,
 		Short:      cliDescription,
 		SuggestFor: []string{"etcdctl"},
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			HandleConfigFile(cmd)
+		},
 	}
 )
 
 func init() {
+	rootCmd.PersistentFlags().StringVarP(&globalFlags.ConfigFile, "config-file", "c", "", "Path to the configuration file for etcdctl. Note that explicitly provided command-line flags and environment variables take precedence.")
```

Comment:
So if a config file is provided or found one in the default location, any other provided command line flags and environment variables be ignored right ? Keeping something inline with what we already have for handling config file for server would be good -  https://github.com/etcd-io/etcd/blob/main/server/etcdmain/config.go#L136. Should also cover that if path provided then default config file will be ignored as well ?